### DISCUSSION TITLE
Export notification incomprehensible

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#61] [Idea retired / denied]
When you start an export you get the following message:

"Your export will be executed
When the export is available we will notify you via our notification service"

German:
"Export wird ausgeführt
Sobald der Export verfügbar ist, informieren wir Dich über unseren Benachrichtigungsdienst."

Message a bit unclear and it's not quite clear what is meant by "notification service".

Maybe you can adjust the text a bit and include the bell symbol? Or alternatively simply offer the export via mail. Email with link when the export is ready.

**Comment by YvesR** [comment]
Message is clear so far, also next update send mails if not read the message in time.
